nilearn/plotting/surf_plotting.py

```diff
@@ -355,7 +370,7 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
         p3dcollec.set_facecolors(face_colors)
 
     if title is not None:
-        plt.title(title)
+        ax.set_title(title, position=(.5, .9))
```

Comment:
Is it necessary that we should set position manually ? Why not leave it to default ?